java-api-core/src/com/kinvey/java/AbstractClient.java

```diff
@@ -463,6 +464,12 @@ public Builder setRequestTimeout(int requestTimeout) {
             return this;
         }
 
+        public Builder setInstanceID(String instanceID) {
+            this.instanceID = instanceID;
+            setBaseUrl(Constants.PROTOCOL_HTTPS + instanceID + Constants.HYPHEN + Constants.HOSTNAME_API);
+            return this;
```

Comment:
Same question as above regarding URL format.

Reply:
`setBaseUrl` also has logic for adding "/" to the end of url.